blueoil/cmd/train.py

```diff
@@ -122,6 +122,12 @@ def start_training(config, profile_step):
 
         saver = tf.compat.v1.train.Saver(max_to_keep=config.KEEP_CHECKPOINT_MAX)
 
+        with open(os.path.join(environment.EXPERIMENT_DIR, "pretrain_vars.txt"), 'w') as pretrain_vars_file:
+            train_vars = tf.compat.v1.trainable_variables()
+            pretrain_vars_file.writelines("[\n")
+            pretrain_vars_file.writelines("\t'%s',\n" % var.name for var in train_vars)
```

Comment:
We will copy and paste this variables. So it's good to use space instead of tab.

Reply:
Thank you. I fixed it.